### PR TITLE
fix(invoice): add WithFeeRule option on invoice

### PR DESCRIPTION
**Pull request by adityaokke**
add with-fee-rule header on request create invoice

### CODE REVIEW

**Review by adityarx — APPROVED** [review]
LGTM! @adityaokke thanks for your contribution. I'm getting a second review for this and then will release this~

**Review by sekaranglila — APPROVED** [review]

**Comment by adityarx** [comment]
Thanks @sekaranglila ! @adityaokke please use v1.0.23

**Pull request merged**
